### ISSUE TITLE
Repl.it Desktop Notifications

### ISSUE THREAD

**Issue by FunnyLamma** [repl-it-discord/repl-it-electron#131]
I am going to work on Desktop notifications, and I would like some help.

**Comment by FunnyLamma** [comment]
I cant find how to send a notification through the app, I need help with this.

**Comment by whippingdot** [comment]
Search online there must be a way. If you can't find it then SOMEONE PLEASE HELP HIM

**Comment by spotandjake** [comment]
Okay pls don't I get emails for every notif. I am gonna help him with them though when I get a minute

**Comment by whippingdot** [comment]
> Okay pls don't I get emails for every notif

If you are talking about github i am really sorry for disturbing you, i will try not to comment from now on.

> I am gonna help him with them though when I get a minute

K cool. Also you guys stopped working on this proj a while back right. You just came back to it 8 days ago? Thanks. I hope you guys continue working on this. IT IS AWESOME SOO FAR.

**Comment by spotandjake** [comment]
Yes we did just come back to it. And I am only saying you do no need to comment on every single one because some of us get emails and some of the comments are just unneeded

**Comment by spotandjake** [comment]
Can look into this, this isnt really a priority though atm so we can work on this one lazily.

**Comment by spotandjake** [comment]
Moving this to next Version do to low importance

**Comment by rxchiie** [comment]
you have probably figured out, but here is a documentation on how to send notifications from electron: https://www.electronjs.org/docs/tutorial/notifications

**Comment by spotandjake** [comment]
Yes we have that part down it is more the parat of watching for notifs from replit

**Comment by Codemonkey51** [comment]
> Yes we have that part down it is more the parat of watching for notifs from replit

They added graphql_subscriptions some time ago, you can prob use chrome network tab to find the exact query, but then it will like send a message through that websocket on every new notification.

**Comment by Codemonkey51** [comment]
> > Yes we have that part down it is more the parat of watching for notifs from replit
> 
> They added graphql_subscriptions some time ago, you can prob use chrome network tab to find the exact query, but then it will like send a message through that websocket on every new notification.

Oops, forgot one part, I belive it will only tell you how many new unread notifs or something like you have a new notif, you would still have to fetch the whole notif's data tho.

**Comment by spotandjake** [comment]
yes lol we know how to do it it i just meant that is what they were referring too when hey said we need to figure it out.
